File: DigitalWare.Models/Factura.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DigitalWare.Models
{
    public class Factura : ClaseBase
    {
        public int Total { get; set; }
        public long VendedorId { get; set; }
        public Vendedor Vendedor { get; set; }
        public List<ProductoFactura> ProductosFacturas { get; set; }
        public long ClienteId { get; set; }
        public Cliente Cliente { get; set; }
    }
}

File: digitalWareAngular/digitalWare/src/app/models/factura.ts
import { claseBase } from "./claseBase";
import { cliente } from "./cliente";
import { productoFactura } from "./productoFactura";
import { vendedor } from "./vendedor";

export class factura extends claseBase {
    total: number;
    vendedorId: number;
    vendedor: vendedor;
    productosFacturas: productoFactura[];
    clienteId: number;
    cliente: cliente;
}
